src/script/strings.ts

```diff
@@ -297,16 +297,16 @@ export const setEntropyStrings = defineMessages({
     id: 'setEntropy.continue',
   },
   headline: {
-    defaultMessage: 'Create Entropy',
+    defaultMessage: 'Increase your account’s security',
```

Comment:
waaaiiiittt I thought this file was autogenerated 🤔  Maybe that's why we don't see the tranlation updates on crowdin....

Reply:
oohh we have something super weird, both are used actually, this is scary. But thanks for the fix 🙏